src/preview.ts

```diff
@@ -31,25 +31,19 @@ const renderToCanvas: RenderToCanvas<RRWebFramework> = async (context, element)
   const response = await fetch(`${url}/${id}`);
   const snapshot = await response.json();
 
-  const iframe = document.createElement('iframe');
-  iframe.setAttribute('style', iframeStyle);
-  element.appendChild(iframe);
+  const htmlNode = snapshot.childNodes[1];
 
   // @ts-expect-error rebuild is typed incorreclty, cache and mirror are optional
-  await rebuild(snapshot, { doc: iframe.contentDocument });
+  const html = (await rebuild(htmlNode, { doc: document })) as HTMLElement;
```

Comment:
I don't think `document` is guaranteed to be an `HTMLElement`. If it happens to be an SVG node, will this still work?

Reply:
@jwir3 sorry I'm not sure I understand. Are you saying that `htmlNode` might actually not be a serialized `html` element, and so when we rebuild it, it might not come out as `HTMLElement`? I'm pretty sure all our snapshots are full documents, so this is guaranteed, but perhaps we should make an assertion to check it?

I'm not sure what we'd do if the assertion failed though.